src/pybind/mgr/dashboard/services/access_control.py

```diff
@@ -368,12 +392,18 @@ def delete_role(self, name):
 
             del self.roles[name]
 
-    def create_user(self, username, password, name, email, enabled=True):
+    def create_user(self, username, password, name, email, enabled=True, pwd_expiry_date=None):
         logger.debug("AC: creating user: username=%s", username)
         with self.lock:
             if username in self.users:
                 raise UserAlreadyExists(username)
-            user = User(username, password_hash(password), name, email, enabled=enabled)
+            # python3
+            # if pwd_expiry_date and (pwd_expiry_date < datetime.timestamp(datetime.utcnow())):
```

Comment:
Should this be removed?

Reply:
I'm not really sure, that's why I only commented it out. I assume it depends of if we do only support Python 3 for the next version or not?